jina/hubble/hubio.py

```diff
@@ -29,12 +39,27 @@ class HubIO:
         - :command:`jina hub pull UUID8` to download the executor identified by UUID8
     """
 
-    def __init__(self, args: 'argparse.Namespace'):
+    def __init__(self, args: "argparse.Namespace"):
         """Create a new HubIO.
         :param args: arguments
         """
```

Comment:
This part needs to be moved under the Class. https://docs1.jina.ai/chapters/docstring/?highlight=docstring#multi-line-docstrings-example-of-a-class